tests/language-feature/namespaces/entrypoint-in-namespace.slang

```diff
@@ -0,0 +1,20 @@
+//TEST:SIMPLE(filecheck=CHECK): -target spirv -entry Pixel.MyType.Main -stage fragment
+
+// Test that we can compile an entrypoint defined in a namespace.
+
+// CHECK: OpEntryPoint
+struct PSInput
+{
+	float4 color : COLOR;
+};
+
+namespace Pixel
+{
+    struct MyType
+    {
+        static float4 Main(PSInput input) : SV_TARGET
+        {
+            return input.color;
+        }
+    }    
+}
```

Comment:
From what I understand this PR also allows us to define entry-points inside a `struct` (but not inside a namespace), should we add that to this test?

Reply:
This test does define the entrypoint inside a struct and a namespace.

Reply:
if things can work for function defined in a struct and a namespace, then it should also work if it is in a struct but not in a namespace, so I don't think we need a separate test for that.

Reply:
```
// My File

struct MyType
{
    static float4 Main(PSInput input) : SV_TARGET
    {
        return input.color;
    }    
}
```